.env

```diff
@@ -1,4 +1,4 @@
-VUE_APP_CLIENT_ID=""
```

Comment:
Don't commit these changes

Reply:
Only base path env variable should be there?